test/ci/CacheAndUpdateITK.sh

```diff
@@ -0,0 +1,54 @@
+#! /usr/bin/env bash
+
+echo ${ExternalData_OBJECT_STORES}
+mkdir -p "${ExternalData_OBJECT_STORES}"
+
+echo "Using ITK repository: ${ITK_REPOSITORY_REMOTE:=https://github.com/InsightSoftwareConsortium/ITK.git}"
```

Comment:
Noting that I switch to using the github ITK repo because is was SIGNIFICANTLY faster that the official itk.org one.